isamples_metadata/isamplesfasttext.py

```diff
@@ -0,0 +1,44 @@
+# Adapted from https://github.com/isamplesorg/vocabulary_learning/blob/develop/fasttext_interface/DwC_predict.py
+
+import fasttext
+import os.path
+import typing
+import re
+
+from fasttext.FastText import _FastText
+
+MODEL_FILENAME = "./sampledFeature.bin"
+THIS_PATH = os.path.dirname(os.path.abspath(__file__))
+MODEL_PATH = os.path.join(THIS_PATH, MODEL_FILENAME)
```

Comment:
@datadavev I found something like this in some `isb` code, so I'm assuming it's a good way to do it.  Let me know if there's a better way.